### PR TITLE
feat(components): [time-picker] add open and close handlers

### PR DESCRIPTION
**Pull request by jagoncalves14**
Add new methods `handleOpen` and `handleClose` to ElTimePicker component, in order to be able to open/close them programmatically.

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by xiaoxian521** [comment]
Hey, thank you very much for your contribution, but here are some highlights that need to be solved by you. We can continue to review the code after solving. One thing to remind is that if the highlight is still not resolved after `10` days, sorry, we will close this pr ❤️

File: docs/en-US/component/time-picker.md [hunk 1]
```diff
@@ -76,7 +76,9 @@ time-picker/range
...
 
-| Method | Description               | Parameters |
-| ------ | ------------------------- | ---------- |
-| focus  | focus the Input component | —          |
-| blur   | blur the Input component  | —          |
+| Method      | Description                  | Parameters |
+| ----------- | ---------------------------- | ---------- |
+| focus       | focus the Input component    | —          |
+| blur        | blur the Input component     | —          |
+| handleOpen  | opens the TimePicker popper  | —          |
```
Comment (xiaoxian521) [inline docs/en-US/component/time-picker.md:83]: use `open` instead of `opens`

File: docs/en-US/component/time-picker.md [hunk 2]
```diff
@@ -76,7 +76,9 @@ time-picker/range
...
+| handleClose | closes the TimePicker popper | —          |
```
Comment (xiaoxian521) [inline docs/en-US/component/time-picker.md:84]: use `close` instead of `closes`

**Review by xiaoxian521 — CHANGES_REQUESTED** [review]

File: docs/en-US/component/time-picker.md (hunk 2 above)
Comment (jagoncalves14) [reply docs/en-US/component/time-picker.md]: Completely agree 👌
I have just committed a fix for this.

File: docs/en-US/component/time-picker.md (hunk 1 above)
Comment (jagoncalves14) [reply docs/en-US/component/time-picker.md]: Completely agree 👌
I have just committed a fix for this.

File: docs/en-US/component/time-picker.md (hunk 2 above)
Comment (xiaoxian521) [reply docs/en-US/component/time-picker.md]: nice

**Review by xiaoxian521 — APPROVED** [review]

**Pull request merged**